microprofile/fault-tolerance-smallrye/extension/src/main/resources/org/wildfly/extension/microprofile/faulttolerance/LocalDescriptions.properties

```diff
@@ -0,0 +1,25 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2019, Red Hat, Inc., and individual contributors
+# as indicated by the @author tags. See the copyright.txt file in the
+# distribution for a full listing of individual contributors.
+#
+# This is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as
+# published by the Free Software Foundation; either version 2.1 of
+# the License, or (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this software; if not, write to the Free
+# Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+# 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+#
+
+microprofile-fault-tolerance-smallrye=MicroProfile Fault Tolerance extension.
+microprofile-fault-tolerance-smallrye.add=Add MicroProfile Fault Tolerance extension.
+microprofile-fault-tolerance-smallrye.remove=Remove MicroProfile Fault Tolerance extension.
```

Comment:
These descriptions describe the subsystem resource, not the extension (which provides the subsystem).
i.e. replace "extension" with "subsystem".

Reply:
Fixed.